src/Tokenizer/Transformer/SquareBraceTransformer.php

```diff
@@ -93,6 +93,19 @@ private function transformIntoDestructuringSquareBrace(Tokens $tokens, $index)
 
         $tokens[$index] = new Token([CT::T_DESTRUCTURING_SQUARE_BRACE_OPEN, '[']);
         $tokens[$endIndex] = new Token([CT::T_DESTRUCTURING_SQUARE_BRACE_CLOSE, ']']);
+
+        $previousMeaningfulIndex = $index;
+        $index = $tokens->getNextMeaningfulToken($index);
+
+        while (null !== $index && $index < $endIndex) {
```

Comment:
Is the first condition (`null !== $index`) possible to achieve? The loop iterates with `getNextMeaningfulToken` and will finally get to position `$endIndex`.